tools/ace-editor/demo/kitchen-sink/docs/python.py

```diff
@@ -4,16 +4,16 @@
 
 # If no arguments were given, print a helpful message
 if len(sys.argv)==1:
-    print '''Usage:
```

Comment:
We should leave the tools/ace-editor code intact, I'm not sure if it even executes with py3..